### ISSUE TITLE
Project creation fails in IDEA with M18 on Windows

### ISSUE THREAD

**Issue by mit-mit** [flutter/flutter-intellij#1383]
## Steps to Reproduce

1. Use Windows
1. Start IDEA, and File>New project, Flutter, App template

![image](https://user-images.githubusercontent.com/13644170/31337508-f17f82ca-acfb-11e7-847c-d95f42639f3e.png)

## Version info

```
Engine • revision 3a12bc092d
Tools • Dart 1.25.0-dev.7.0

C:\Users\mit\dev\github\flutter>flutter doctor
[√] Flutter (on Microsoft Windows [Version 10.0.10586], locale en-US, channel master)
    • Flutter at C:\Users\mit\dev\github\flutter
    • Framework revision 838bd63182 (3 months ago), 2017-07-25 16:39:03 -0700
    • Engine revision 3a12bc092d
    • Tools Dart version 1.25.0-dev.7.0

[√] Android toolchain - develop for Android devices (Android SDK 25.0.3)
    • Android SDK at C:\Users\mit\AppData\Local\Android\sdk
    • Platform android-25, build-tools 25.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_112-release-b06)

[√] Android Studio (version 2.3)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Java version OpenJDK Runtime Environment (build 1.8.0_112-release-b06)

[-] IntelliJ IDEA Community Edition (version 2017.2)
    X Flutter plugin not installed; this adds Flutter specific functionality.
    • Dart plugin version 172.4155.35
    • For information about installing plugins, see
      https://flutter.io/intellij-setup/#installing-the-plugins

[√] Connected devices
    • None


C:\Users\mit\dev\github\flutter>


```

**Comment by mit-mit** [comment]
cc @devoncarew @stevemessick

**Comment by stevemessick** [comment]
Looks like the Flutter SDK is 3 months old. I doubt the --template option existed then.

**Comment by mit-mit** [comment]
Hmm, I thought I had run upgrade. Let me try again!

**Comment by mit-mit** [comment]
So sorry, working fine after upgrade
